pkg/server/server.go

```diff
@@ -67,6 +69,7 @@ func New(opt Options) (*Server, error) {
 	}
 
 	s := grpc.NewServer(sOpts...)
+	grpclog.SetLogger(NewLogger(os.Stdout))
 
 	us := user.NewService(uOps)
 	us.RegisterService(s)
```

Comment:
Makes senses, but, I disagree with toil.
Change elb configs by hand is so annoying and isn't multi-cloud.